README.md

```diff
@@ -1 +1,74 @@
-# AMiDEAST
\ No newline at end of file
+# AMiDEAST
+
+AMIDEAST is an website that allows the staff members add and view courses,it can also mange students,in the other hands the student can see the proposed courses and select one to apply for.
```

Comment:
```suggestion
AMIDEAST is a website that allows the staff members to add and view courses, it can also manage students, in the other hands the student can see the proposed courses and select one to apply for.
```